src/ui/window.ui

```diff
@@ -1,279 +1,153 @@
-            <property name="margin-bottom">20</property>
             <property name="orientation">vertical</property>
             <child>
               <object class="GtkScrolledWindow" id="treeview_scrolled_window">
-                <property name="can-focus">True</property>
-                <property name="shadow-type">in</property>
-                <child>
+                <property name="child">
                   <object class="GtkTreeView" id="treeview">
-                    <property name="visible">True</property>
-                    <property name="can-focus">True</property>
-                    <property name="hexpand">True</property>
-                    <property name="vexpand">True</property>
-                    <property name="rules-hint">True</property>
+                    <property name="hexpand">1</property>
+                    <property name="vexpand">1</property>
                     <child internal-child="selection">
                       <object class="GtkTreeSelection"/>
                     </child>
                   </object>
-                </child>
+                </property>
               </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="position">1</property>
-              </packing>
             </child>
           </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="position">2</property>
-          </packing>
         </child>
         <child>
           <object class="GtkLabel" id="save_info_label">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
+            <property name="margin-top">20</property>
             <property name="margin-bottom">20</property>
           </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="position">3</property>
-          </packing>
         </child>
       </object>
     </child>
     <child type="titlebar">
       <object class="GtkHeaderBar" id="headerbar">
-        <property name="visible">True</property>
-        <property name="can-focus">False</property>
-        <property name="title">Curtail</property>
-        <property name="subtitle" translatable="yes">Compress your images</property>
-        <property name="show-close-button">True</property>
         <child>
-          <object class="GtkButtonBox">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
-            <property name="homogeneous">True</property>
-            <property name="layout-style">expand</property>
+          <object class="GtkBox">
+            <style>
+              <class name="linked"/>
```

Comment:
Dont do this, see https://blogs.gnome.org/alexm/2021/09/10/cleaning-up-header-bars/